README.md

```diff
@@ -0,0 +1,44 @@
+# My Porfolio
```

Comment:
Please use all the files from [this](https://github.com/microverseinc/linters-config/tree/master/html-css) linter config.

Reply:
Hello Sajjad, please I'm new to this, could you explain what I need to do. I'm some how confused. Thanks for your understanding